### PR TITLE
Muriel Colors: update FancyButtons

### PR DESCRIPTION
**Pull request by nheagy**
Refs #11683

This updates the FancyButtons to match the muriel colors.

| Flag Off/Old Colors | Flag On/Muriel |
| ---- | ---- |
| Simulator Screen Shot - iPhone Xʀ - 2019-06-27 at 15 29 18 | Simulator Screen Shot - iPhone Xʀ - 2019-06-27 at 15 23 37 |

### Reference mockup:

Screen Shot 2019-06-26 at 2 22 31 PM

**Note:** The secondary colors were so similar between the two, I didn't have to add new semantic colors for them. But since the primary is blue vs. pink, I did for those.

To test:
- Check out the login epilogue screen
- And a `no results view`
- Also a FancyAlert, like when you login for the first time on a fresh install
- Ensure the buttons are the proper colors

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by nheagy** [comment]
@ScoutHarris mind taking a look, since I also cut up `NoResultsViewController` a bit?

File: WordPress/Classes/Extensions/UIColor+MurielColors.swift [hunk 1]
```diff
@@ -247,4 +247,34 @@ extension UIColor {
             return WPStyleGuide.greyLighten10()
         }
     }
+
+    /// MARK: Muriel colors for buttons
+    static var buttonBase: UIColor {
```
Comment (frosty) [inline WordPress/Classes/Extensions/UIColor+MurielColors.swift]: Should we give these types of button a specific name? There may be more types of button! `primaryButton`? Or do we address that if necessary later?

File: WordPress/Classes/Extensions/UIColor+MurielColors.swift [hunk 2]
```diff
@@ -247,4 +247,34 @@ extension UIColor {
...
+        if FeatureFlag.murielColors.enabled {
+            return .accent
+        } else {
+            return WPStyleGuide.mediumBlue()
+        }
+    }
+    static var buttonBorder: UIColor {
```
Comment (frosty) [inline WordPress/Classes/Extensions/UIColor+MurielColors.swift]: Nitpick, but could we add a blank line space between each of these `var`s?

**Comment by frosty** [comment]
Forgot to add, this pink looks awesome! 😍

File: WordPress/Classes/Extensions/UIColor+MurielColors.swift (hunk 1 above)
Comment (nheagy) [reply WordPress/Classes/Extensions/UIColor+MurielColors.swift]: I'm not sure on this one!

**Comment by nheagy** [comment]
> The NUX flow still has the old nav bar colors etc too

Ya, it has a separate config for the whole flow. That's one of the things I'll be doing next. I'll add it to the master issue.

**Comment by nheagy** [comment]
@frosty ready for another look 😀

**Pull request merged**